Gemfile.lock

```diff
@@ -499,38 +502,38 @@ GEM
     omniauth-rails_csrf_protection (1.0.1)
       actionpack (>= 4.2)
       omniauth (~> 2.0)
-    pagy (3.6.0)
-    parallel (1.19.2)
-    parallel_tests (2.21.1)
+    pagy (3.14.0)
+    parallel (1.24.0)
+    parallel_tests (2.32.0)
       parallel
     parser (2.7.2.0)
       ast (~> 2.4.1)
-    path_expander (1.0.4)
-    pg (1.1.3)
-    pry (0.10.3)
-      coderay (~> 1.1.0)
-      method_source (~> 0.8.1)
-      slop (~> 3.4)
+    path_expander (1.1.1)
```

Comment:
### [path_expander](https://github.com/seattlerb/path_expander)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.0.4 → 1.1.1

[_~~change-log~~, ~~source-code~~, [gem-diff](https://my.diffend.io/gems/path_expander/1.0.4/1.1.1)_]





<details>
<summary>Commits</summary>

A change of **6** commits. See the full changes on [the compare page](https://github.com/seattlerb/path_expander/compare/v1.0.4...v1.1.1).

These are the individual commits:
- (bb800d3) [- Fixed a deprecation warning from File.exists?](https://github.com/seattlerb/path_expander/commit/bb800d3d8d65a5eda14ba100f515dad3eff513da)
- (4f742da) [prepped for release](https://github.com/seattlerb/path_expander/commit/4f742da80f919c17bd89bb0bc4ddafb68c438a78)
- (4743c0b) [+ Added a default path (default: ".") to scan if no files ar…](https://github.com/seattlerb/path_expander/commit/4743c0bc497dbf29f1efcb6b9d7717948110a51c)
- (77cd258) [prepped for release](https://github.com/seattlerb/path_expander/commit/77cd258a7f06361f776c7e44fe5e4c217e2ddc0a)
- (3aa0df8) [- Process plain '-' as a file, not an option.](https://github.com/seattlerb/path_expander/commit/3aa0df8e5a2013e9773ac00f83faaa6e88397349)
- (993586e) [prepped for release](https://github.com/seattlerb/path_expander/commit/993586e68fcb30e3a83edb37ba0b6bf7914457c0)


</details>